hack/generate.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -eux
+
+# Generate jsonnet mixin prometheusrule manifest.
+
+cd jsonnet && jb update && jsonnet -J vendor main.jsonnet  | gojsontoyaml > ../manifests/0000_90_etcd-operator_03_prometheusrule.yaml
```

Comment:
question: could we look into adding this to github.com/openshift/build-machinery-go so the rest of the control-plane can emulate this process if they would like. Then also it will become part of our Makefile workflow? `make update`

Reply:
Yes that sound good, if you don't mind I would do that in a follow-up. 

Note that this would only be used if we fix things upstream or once a new release is brought in.

Reply:
should we add a linter test as part of the process? Is jsonnet-lint useful

Reply:
I use jsonnetfmt in my vim so it formats everything correctly, but haven't tried jsonnet-lint. Can look into it.